src/main/frontend/db/rules.cljs

```diff
@@ -7,11 +7,17 @@
      [?c :block/parent ?t]
      (parent ?p ?t)]
```

Comment:
it looks 'parent' rule can be deleted

Reply:
Just keep it in case we need child -> parent queries in future.
I'm adding some descriptions to this

Reply:
@cnrpman Agreed. How about a comment above child and parent so everyone knows what each rule is optimized for?